acceptance/tests/base/dsl/helpers/host_helpers/archive_file_from_test.rb

```diff
@@ -19,17 +19,20 @@
     assert_equal(true, Dir.exists?('archive/sut-files'))
   end
 
-  step "file is copied to local <archiveroot>/<host name> directory" do
+  step "archive is copied to local <archiveroot>/<host name> directory" do
```

Comment:
This step title indicates that it will write it to hostname directory, and I see the code that adds it in the `archive_file_from` method, but I don't see verification of the hostname in the filepath.